extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -106,7 +113,14 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         .getVisibilityState();
     const adTestOn = this.element.getAttribute('data-adtest') ||
         isInManualExperiment(this.element);
-    const format = `${slotRect.width}x${slotRect.height}`;
+    const width = slotRect.width;
```

Comment:
Agree that you modified to use height/width attributes when provided (which makes me now wonder whether responsive works correctly for type=doubleclick slots) however Adsense is still deriving format based on slot rect.

@bobcassels was previously looking into issue that AMP intersection observer was incorrectly including border causing width/height to be incorrect.  Bob did you ever fix?